apis/v1alpha1/defaulting.go

```diff
@@ -151,6 +151,13 @@ func (in *GreptimeDBCluster) defaultSpec() *GreptimeDBClusterSpec {
 
 		// Set the default logging format to JSON if monitoring is enabled.
 		defaultSpec.Logging.Format = LogFormatJSON
+
+		// Turn on the slow query log by default if monitoring is enabled.
+		defaultSpec.Logging.SlowQuery = &SlowQuery{
+			Enabled:     true,
+			Threshold:   "10s",
```

Comment:
Question: Enabling this by default affect GreptimeDB performance?

Reply:
Most of time, it would not affect the performance. Also, we can use `sampleRatio` to control it.